springfox-core/src/main/java/springfox/documentation/service/AllowableListValues.java

```diff
@@ -37,4 +39,25 @@ public AllowableListValues(List<String> values, String valueType) {
   public String getValueType() {
     return valueType;
   }
+  
+  @Override
+  public int hashCode() {
+    return Objects.hashCode(values, valueType);
+  }
+  
+  @Override
+  public boolean equals(Object o) {
+    if (this == o) {
+      return true;
+    }
+
+    if (o == null || getClass() != o.getClass()) {
+      return false;
+    }
+
+    AllowableListValues that = (AllowableListValues) o;
+
+    return Objects.equal(values, that.values) &&
+        Objects.equal(valueType, that.valueType);
+  }
```

Comment:
Same as above. I'd rather extract this to an `Equivalence` [like this](https://github.com/springfox/springfox/blob/master/springfox-spring-web/src/main/java/springfox/documentation/spring/web/OperationCachingEquivalence.java).